src/ContentHubClient.php

```diff
@@ -255,6 +255,9 @@ public static function register(
    *
    * @return bool
    *   Whether the clientName from the request matches the name passed to it.
+   *
+   * @deprecated in 3.7.0 and is removed from 3.8.0. Use getClientByName().
+   * @see getClientByName()
```

Comment:
I think this message should be `@deprecated in 3.6.1 and ....`
If not then, PR should be made for 3.7.x  and not for 3.6.x.

@kirtigarg2584 @saxumVermes what you guys think? 🤔